inst/odin/carehomes.R

```diff
@@ -1579,6 +1579,37 @@ new_sympt_cases_non_variant_over25 <-
 update(cum_sympt_cases_non_variant_over25) <-
   cum_sympt_cases_non_variant_over25 + new_sympt_cases_non_variant_over25
 
+## new pillar 2 by age
+initial(cum_sympt_cases_under15) <- 0
+new_sympt_cases_under15 <- sum(n_EI_P[1:3, , ])
+update(cum_sympt_cases_under15) <- cum_sympt_cases_under15 +
+  new_sympt_cases_under15
+
+initial(cum_sympt_cases_15_24) <- 0
+new_sympt_cases_15_24 <- sum(n_EI_P[4:5, , ])
+update(cum_sympt_cases_15_24) <- cum_sympt_cases_15_24 +
+  new_sympt_cases_15_24
+
+initial(cum_sympt_cases_25_49) <- 0
+new_sympt_cases_25_49 <- sum(n_EI_P[6:10, , ]) + (sum(n_EI_P[18, , ]) / 8) * 5
+update(cum_sympt_cases_25_49) <- cum_sympt_cases_25_49 +
+  new_sympt_cases_25_49
+
+initial(cum_sympt_cases_50_64) <- 0
+new_sympt_cases_50_64 <- sum(n_EI_P[11:13, , ]) + (sum(n_EI_P[18, , ]) / 8) * 3
+update(cum_sympt_cases_50_64) <- cum_sympt_cases_50_64 +
+  new_sympt_cases_50_64
+
+initial(cum_sympt_cases_65_79) <- 0
+new_sympt_cases_65_79 <- sum(n_EI_P[14:16, , ]) + (sum(n_EI_P[19, , ]) * 0.25)
+update(cum_sympt_cases_65_79) <- cum_sympt_cases_65_79 +
+  new_sympt_cases_65_79
+
+initial(cum_sympt_cases_80_plus) <- 0
+new_sympt_cases_80_plus <- sum(n_EI_P[17, , ]) + (sum(n_EI_P[19, , ]) * 0.75)
```

Comment:
again maybe comment explaining the splitting of groups 18/19

Reply:
done